Aptoide/aptoide-client-v8
# [feature/ASV-1415] Curation more than 1 app

**What does this PR do?**

   This PR enables the view to handle more than one app coming from the API, by removing the animation and the bottom app card in those cases.
  In the first commit all the Editorial related files were moved to a specific Editorial package

**Database changed?**

No

**Where should the reviewer start?**

EditorialService.java
EditorialViewModel.java
EditorialContent.java
EditorialFragment.java
EditorialViewHolder.java

**How should this be manually tested?**

Mock Charles 7.xxx card id response so that 
1. Doesn’t return any apps, and check if it works
2. Returns more than 1 app (see getApp request to copy the specification of the file as some flows won’t be possible if it’s the same app). Check navigation to app view (click on card) and check all the install flows (both installing, cancel, pause, unpause)

3 (No need to mock) Check if the one app flow works as before (Animation working, click on placeholder and bottom card to navigate, check the previous install flows on both the placeholder and bottom card)
**What are the relevant tickets?**

Tickets related to this pull-request: 

https://aptoide.atlassian.net/browse/ASV-1415


**Code Review Checklist**
